phpunit.xml

```diff
@@ -14,8 +14,7 @@
 			<directory prefix="tests-" suffix=".php">./tests/unit/legacy</directory>
 		</testsuite>
 		<testsuite name="Unit Tests">
-			<directory suffix="Test.php">./tests/unit/tests</directory>
-            <directory prefix="Test" suffix=".php">./tests/unit/tests</directory>
+            <directory prefix="Test" suffix="php">./tests/unit/tests</directory>
```

Comment:
I believe we want to support both the suffix and prefix, as some of our tests are one way and some are the other. Cleaning that up will be another task.